docs/reference/vectors/vector-functions.asciidoc

```diff
@@ -8,6 +8,16 @@ linearly scanned. Thus, expect the query time grow linearly
 with the number of matched documents. For this reason, we recommend
 to limit the number of matched documents with a `query` parameter.
 
+This is the list of available vector functions:
+
+1. `cosineSimilarity` – calculates cosine similarity
+2. `dotProduct` – calculates dot product
+3. `l1norm` – calculates L^1^ distance
+4. `l2norm` - calculates L^2^ distance
+5. `getVectorValue` – returns a vector's value as an array of floats
```

Comment:
It looks like painless will expose these methods as `doc['field'].vectorValue` and `doc['field'].vectorMagnitude`. Should we emphasize that usage pattern instead of calling getters directly? Also I wonder if we could rename `vectorMagnitude` to `magnitude` for simplicity.

Reply:
That's very correct,  using just `vectorValue` and `magnitude` access methods are much better. Addressed in 388b1f85029214d3b8cd53f5f2ed5aec2023f6f3